purchases/src/main/kotlin/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -66,6 +70,32 @@ internal class IdentityManager(
         }
     }
 
+    suspend fun aliasOldUserIdToCurrentOne(
+        oldAppUserID: String,
+    ) {
```

Comment:
Naming suggestion: 
```suggestion
    suspend fun aliasCurrentUserIdTo(
        oldAppUserID: String,
    ) {
```
  
  
I like how that looks on the call site, but I realize that's subjective so definitely not a blocker.
```kotlin
identityManager.aliasCurrentUserIdTo(oldUserId)
```